jobs/nats/monit

```diff
@@ -3,3 +3,13 @@ check process nats
   start program "/var/vcap/jobs/bpm/bin/bpm start nats"
   stop program "/var/vcap/jobs/bpm/bin/bpm stop nats"
   group vcap
+
+check process bosh_nats_sync
+  with pidfile /var/vcap/sys/run/bpm/nats/bosh_nats_sync.pid
+  start program "/var/vcap/jobs/bpm/bin/bpm start nats -p bosh_nats_sync"
+  stop program "/var/vcap/jobs/bpm/bin/bpm stop nats -p bosh_nats_sync"
+  group vcap
+
+check file nats_auth_conf
```

Comment:
This is hard to do in any other way because of the usage of BPM. We cannot restart NATS from the bosh_nats_sync process because both of them are sandboxed. Once we replace Monit with something else we need to use that tool to restart NATS as well.